### PR TITLE
Document posting, update post schema and delete unused file that causes Docusaurus to warn

### PR DESCRIPTION
**Pull request by NotFenixio**

### CODE REVIEW

File: docs/rest-api/intro/posting.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+The endpoint for posting into home is `/home`, sending a POST request with this body:
+```json
+{
+    "content": "Post content here"
+}
+```
+
+For posting in livechat or group chats, you'll only need to change the endpoint to `/posts/livechat` or `/posts/CHAT_ID`, respectively.
+
+In exchance, you'll get a [post object](/objects/post#examples) that looks like this:
```
Comment (mybearworld) [inline docs/rest-api/intro/posting.md]:
```suggestion
In exchange, you'll get a [post object](/objects/post#examples) that looks like this:
```

File: docs/rest-api/intro/posting.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+```json
```
Comment (mybearworld) [inline docs/rest-api/intro/posting.md:20]: Does this JSON blob need to be here? The shape is already explained on the post object page.

File: docs/rest-api/intro/posting.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+slug: /rest-api/intro/posting
+sidebar_label: Posting
+---
+
+# Posting
+:::note
+This endpoint requires [authentication](/rest-api/intro/authentication).
+:::
+
+The endpoint for posting into home is `/home`, sending a POST request with this body:
```
Comment (mybearworld) [inline docs/rest-api/intro/posting.md:11]: Could the post size restrictions and how posts can't only consist of whitespace be explained here?

File: docs/rest-api/intro/posting.md (hunk 2 above)
Comment (NotFenixio) [reply docs/rest-api/intro/posting.md:20]: Yeah it probably doesn't.

**Comment by showierdata9978** [comment]
tbh i prefer JSON blobs on api documentation as it's easier to copy for types n stuff

File: docs/rest-api/intro/posting.md (hunk 3 above)
Comment (NotFenixio) [reply docs/rest-api/intro/posting.md:11]: Sure.

**Comment by mybearworld** [comment]
> tbh i prefer JSON blobs on api documentation as it's easier to copy for types n stuff

It should probably still be on the Post Object page, not on every single page using a post object

**Comment by NotFenixio** [comment]
This is ready to merge, by the way.

**Pull request merged**